deployment_cdk/_layout_deployment.adoc

```diff
@@ -44,7 +44,7 @@ endif::production_build[]
 
 == Deployment options
```

Comment:
```suggestion
== Choose a deployment option
```
This updated heading reflects the new actionable nature of the content.

(Also applies to the CFN, EKS, and Terraform versions of `_layout_deployment.adoc`.)